docs/api-reference/layer.md

```diff
@@ -402,23 +402,32 @@ new Layer({
       duration: 300,
       easing: d3.easeCubicInOut,
       enter: value => [value[0], value[1], value[2], 0] // fade in
+    },
+    getRadius: {
+      type: 'spring',
+      stiffness: 0.01,
+      damping: 0.15,
+      enter: value => [0] // grow from size 0
     }
   }
 });
 ```
 
 | Parameter     | Type       | Default     | Description |
 | ---------     | --------   | ----------- | ----------- |
-| `duration`    | `Number`   | `0`         | Duration of the transition animation, in milliseconds |
-| `easing`      | `Function` | LINEAR (`t => t`) | Easing function that maps a value from [0, 1] to [0, 1], see [http://easings.net/](http://easings.net/)  |
-| `enter`      | `Function` | APPEARANCE (`value => value`) | Callback to get the value that the entering vertices are transitioning from. See notes below |
+| `type`        | `String`   | `interpolation` | Type of the transition (either `interpolation` or `spring`) |
+| `duration`    | `Number`   | `0`         | Duration of the transition animation, in milliseconds (only for `interpolation` transition `type`) |
```

Comment:
This is getting hard to read. Should we split these into separate tables, sorted by `type`?